### ISSUE TITLE
how to insert my own image ?

### ISSUE THREAD

**Issue by Jagan3534** [x4nth055/pythoncode-tutorials#77]
how can i insert my image inside the code ,
im unable to fine the way please help 

thank you sir

**Comment by c0d33ngr** [comment]
Yes... how can i load and detect with my own images and not from datasets too
